src/common/IOSUtils.ts

```diff
@@ -269,7 +269,11 @@ export class IOSUtils {
         projectDir: string,
         appBundlePath: string | undefined,
         targetApp: string,
-        targetAppArguments: LaunchArgument[]
+        targetAppArguments: LaunchArgument[],
+        // tslint:disable-next-line: no-unnecessary-initializer
```

Comment:
Sorry, it's not clear to me: why *do* we need to initialize these values to undefined?

Reply:
Unless we have a desire to let these be optional arguments (which we may?), we should just change the tests to explicitly pass in `undefined`. The only use place we consume this method, explicitly delineates its own values.

Reply:
sure, makes sense